### ISSUE TITLE
Apply sigmoid to fused BB25 scores

### ISSUE THREAD

**Issue by davidmezzetti** [neuml/txtai#1053]
**Update: I tested applying sigmoid to fused scores and confirmed the same BEIR scores, going to make this change**

Hi @jaepil 

Quick question on LogOdds normalization. Most people are used to scores between 0 - 1 but LogOdds is returning raw logits (https://github.com/neuml/txtai/blob/master/src/python/txtai/embeddings/search/hybrid.py#L120)

What was the downside of converting these logits back through sigmoid? I'd at least prefer an option to do it but I wanted to ask before changing.
